### DISCUSSION TITLE
Need to take backup of k8s in the local directory or NFS directory

### DISCUSSION THREAD

**Discussion by avi-git-lab** [vmware-tanzu/velero#6362] [Community support Q&A]
Dear members, In our project we require to take backup of k8s volumes in one of the directory either on the local file system or in the shared NFS directory. We don't want to use minio for this. Kindly let me know how to perform this. Can we turn NFS into S3-like filesystem and use it Or can we turn local host directory in S3 like bucket?

**Comment by farazkhawaja** [comment]
bump @sseago

**Reply by kaovilai** [reply]
https://github.com/replicatedhq/local-volume-provider

**Comment by Lyndon-Li** [comment]
Please vote on issue #1229 and share your use case of why NFS is required over minio
